static/lookupConfig.json

```diff
@@ -930,11 +925,6 @@
     "uri": "https://id.loc.gov/vocabulary/msoundcontent",
     "component": "list"
   },
-  {
-    "label": "special playback characteristics",
-    "uri": "https://id.loc.gov/vocabulary/mplayback",
-    "component": "list"
-  },
```

Comment:
this isn't a duplicate -- is it unused??

Reply:
It's a duplicate of https://github.com/LD4P/sinopia_editor/blob/dc995f042c1452872bbbeb077b2546826415d0a5/static/lookupConfig.json#L900

Reply:
See the issue for context: https://github.com/LD4P/sinopia_editor/issues/1035#issuecomment-514347925

Reply:
gotcha - thanks.